src/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  Breadcrumbs as MuiBreadcrumbs,
+  breadcrumbsClasses
+} from "@mui/material";
+
+import { BreadcrumbsProps } from "./Breadcrumbs.types";
+import NavigateNextIcon from "@mui/icons-material/NavigateNext";
+import React from "react";
+
+/**
+ * Breadcrumbs component
+ */
+function Breadcrumbs({ children, ...rest }: BreadcrumbsProps) {
+  return (
+    <MuiBreadcrumbs
+      {...rest}
+      aria-label="breadcrumbs"
+      separator={<NavigateNextIcon fontSize="small" />}
```

Comment:
Storybook is unable to detect this as the default value in the generated docs.

![image](https://github.com/IPG-Automotive-UK/react-ui/assets/27866636/25b53e5d-3f83-43fb-b3cb-bd79f86cabb0)

Seems like the following syntax works:

```suggestion
function Breadcrumbs({
  children,
  separator = <NavigateNextIcon fontSize="small" />,
  ...rest
}: BreadcrumbsProps) {
  return (
    <MuiBreadcrumbs
      {...rest}
      aria-label="breadcrumbs"
      separator={separator}
```

![image](https://github.com/IPG-Automotive-UK/react-ui/assets/27866636/afa31fbc-ffee-4824-9e00-6e65522a7c91)